src/lenskit/metrics/_base.py

```diff
@@ -22,27 +25,76 @@ class Metric(ABC):
     Base class for LensKit metrics.  Individual metrics need to implement a
     sub-interface, such as :class:`ListMetric` and/or :class:`GlobalMetric`.
 
-    For simplicity in the analysis code, you cannot simply implement the
-    properties of this class on an arbitrary class in order to implement a
-    metric with all available behavior such as labeling and defaults; you must
-    actually extend this class.  This requirement may be relaxed in the future.
+    This class defines the interface for metrics. Subclasses should implement
+    the `measure_list` method to compute metric values.
+
+    The `summarize()` method has a default implementation that computes the
+    mean of the per-list metric values, but subclasses can override it to provide
+    more appropriate summary statistics.
 
     Stability:
         Full
+
+    .. note::
+
+        For simplicity in the analysis code, you cannot simply implement the
+        properties of this class on an arbitrary class in order to implement a
+        metric with all available behavior such as labeling and defaults;
+        you must actually extend this class. This requirement may be relaxed
+        in the future.
+    """
+
+    default: ClassVar[float | None] = 0.0
+    """
+    The default value to infer when computing statistics over missing values.
+    If ``None``, no inference is done (necessary for metrics like RMSE, where
+    the missing value is theoretically infinite).
```

Comment:
"infer" -> "impute", "inference" -> "imputation".

Reply:
Should `default` live on `ListMetric` instead of `Metric`? A single `default` value for a metric, that may compute more than one metric, doesn't seem to make sense.